jujugui/static/gui/src/test/test_app.js

```diff
@@ -693,7 +693,7 @@ describe('App', function() {
           assert.strictEqual(
             node.props.helpMessage,
             'Find your username and password with ' +
-            '`juju show-controller --include-passwords`'
+            '`juju show-controller --show-password`'
```

Comment:
How did this old test pass when it had `--include-passwords` and not the old `--show-passwords`?